Marlin/src/lcd/ultralcd.cpp

```diff
@@ -3478,6 +3575,38 @@ void lcd_quick_feedback(const bool clear_buttons) {
           MENU_MULTIPLIER_ITEM_EDIT(int3, MSG_EXTRA_FAN_SPEED " 3", &new_fanSpeeds[2], 3, 255);
         #endif
       #endif
+      END_MENU();
+    }
+  #endif
+
+  /**
+   *
+   * "Temperature" submenu
+   *
+   */
+  void lcd_heaters_menu() {
+    START_MENU();
+    MENU_BACK(MSG_BACK);
+
+    //
+    // Nozzles:
+    // Nozzle [1-5]:
+    //
+    #if HOTENDS > 0
+      MENU_ITEM(submenu, MSG_NOZZLE "s", lcd_nozzle_temp_menu);
```

Comment:
This is not translation compatible because different languages use different methods to make words plural.